src/qml/Tracking.qml

```diff
@@ -179,7 +179,8 @@ Item{
                 onApply: {
                     if( Number(timeIntervalText.text) + Number(distanceText.text) === 0 ||
                         ( timeIntervalCheck.checked && distanceCheck.checked && conjunction.checked &&
-                          ( Number(timeIntervalText.text) === 0 || Number(distanceText.text) === 0 ) ) )
+                          ( Number(timeIntervalText.text) === 0 || Number(distanceText.text) === 0 ) ) ||
+                        !( timeIntervalCheck.checked || distanceCheck.checked ) )
```

Comment:
```suggestion
                        ( !timeIntervalCheck.checked && !distanceCheck.checked ) )
```

I think this is equal? Is it just me who finds it difficult to read negations on complex conditions?

Reply:
It doesn't matter for me. I found it more understandable, because that's what you would explain: "Do not start if not one of them is checked" but no strong opinion.